### PR TITLE
Refresh CI/CD page

### PR DESCRIPTION
**Pull request by chrsmith**
Refresh the Continuous Delivery page to use logos of CI/CD providers we have guides for, as well as make the page read a lot better. Cut a few paragraphs that weren't adding much. Similarly we were plugging the Pulumi GitHub App like 3-times on the same page.

I couldn't figure out how to get things to work right using tailwind, and even then I didn't like the duplication. (And need for minor tweaks to some logos which weren't sized like the others.) So in the end I just created some dedicated CSS.

New logo grid at different sizes:
image
image
image

### CODE REVIEW

**Comment by joeduffy** [comment]
Not pertaining to your change, but it's interesting the GitHub Action didn't run. It appears your branch has a base before my commit last night. Just a tidbit to stuff away in the "How GitHub Actions Work" department (they don't use the definitions in `master`).

File: assets/sass/styles.scss [hunk 1]
```diff
@@ -129,3 +129,46 @@ main {
...
 
 @tailwind utilities;
+
+// Widget showing a nicely formatted logo grid of our supported CI/CD platforms.
+.supported-cicd-platforms {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: space-around;
+    align-content: space-around;
```
[the remark this answers is not in the document]
Comment (justinvp) [reply assets/sass/styles.scss]: Note that the nice thing about doing this is it inherits the `btn` styles, so you get nicer transitions on the background change on hover.

File: assets/sass/styles.scss [hunk 2]
```diff
@@ -129,3 +129,46 @@ main {
 }
 
 @tailwind utilities;
+
+// Widget showing a nicely formatted logo grid of our supported CI/CD platforms.
+.supported-cicd-platforms {
```
[the remark this answers is not in the document]
Comment (chrsmith) [reply assets/sass/styles.scss]: Done.

File: assets/sass/styles.scss (hunk 1 above)
[the remark this answers is not in the document]
Comment (chrsmith) [reply assets/sass/styles.scss]: Done. I had seen `@apply` elsewhere but was unaware of what it actually _did_. Thanks for taking the time to rework this, sorry for the trouble.

**Pull request merged**